src/main/resources/GeppettoCommonLibrary.xmi

```diff
@@ -55,4 +55,10 @@
   <types xsi:type="gep_1:MetadataType"
       id="metadata"
       name="Metadata"/>
+  <types xsi:type="gep_1:ConnectionType"
+      id="edge"
```

Comment:
What about additional fields? ![image](https://user-images.githubusercontent.com/46561561/67492405-a701d600-f676-11e9-8c83-398f4ed50a09.png)

Reply:
I thought we can put them in the Value

Reply:
I was thinking as default "class" values. Name is clearly not something we can put there though. Probably this os something we can add to types on the application directly

Reply:
Yes I was thinking that unless we are customizing them it's not needed even to add them and if we are customizing them then we can do that in other libraries or in the application